source/core/ut_utils.pks

```diff
@@ -22,7 +22,11 @@ create or replace package ut_utils authid definer is
    */
 
   gc_version                 constant varchar2(50) := 'v3.1.7.2844-develop';
-
+  
+  gc_invalid_first_xml_char  constant varchar2(50)  := '[^_a-zA-Z]';
```

Comment:
Those should probably be private constants.

Reply:
Done